Makefile

```diff
@@ -153,13 +153,27 @@ docker-test-unit: $(DOCKER_CACHE)
 	@$(DOCKER_ENV) make test-unit
 	@chmod -R 0755 $<
 
-.PHONY: docker-test-integration
-docker-test-integration: .coverage
+.PHONY: build-test-image
+build-test-image:
 	@echo "===> Building Antrea Integration Test Docker image <==="
 ifneq ($(NO_PULL),)
 	docker build -t antrea/test -f build/images/test/Dockerfile $(DOCKER_BUILD_ARGS) .
 else
 	docker build --pull -t antrea/test -f build/images/test/Dockerfile $(DOCKER_BUILD_ARGS) .
+endif
+
+HAS_TEST_IMAGE = $(shell docker images antrea/test | grep antrea/test > /dev/null && echo "yes")
+.PHONY: docker-test-integration
+docker-test-integration: .coverage
+ifneq ($(CHECK),true)
+	@echo "===> You can set 'CHECK=true' to check and use existing image if it does exist <==="
```

Comment:
Why such a generic variable name for this?

Reply:
I was thinking it's better to give it a short name to have less input, rename it to `CHECK_IMAGE`